src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -282,6 +284,9 @@ private ICitizenData spawnCitizenOnPosition(
         entity.setPosition(spawnPoint.getX() + HALF_BLOCK, spawnPoint.getY() + SLIGHTLY_UP, spawnPoint.getZ() + HALF_BLOCK);
         world.addEntity(entity);
 
+        colony.getEventDescriptionManager().addEventDescription(citizenData.isChild() ? new CitizenBornEvent(spawnPoint, citizenData.getName()) :
```

Comment:
Yeah you will need to modify this.
This call location is less then appropriate.

Find the code that actually triggers the birth and fire it from there.

Reply:
Addons already told me this, i will fix this once i continue working on this pr later today.

Reply:
done.